packages/plugins/Gfi/README.md

```diff
@@ -25,6 +25,40 @@ The GFI plugin can be used to fetch and optionally display GFI (GetFeatureInfo)
 
 For details on the `displayComponent` attribute, refer to the [Global Plugin Parameters](../../core/README.md#global-plugin-parameters) section of `@polar/core`.
 
+<details>
+<summary>Example configuration</summary>
+
+```js
+gfi: {
+  mode: 'bboxDot',
+  activeLayerPath: 'plugin/layerChooser/activeMaskIds',
+  layers: {
+    [denkmaelerWFS]: {
+      geometry: true,
+      window: true,
+      maxFeatures: 10,
+      geometryName: 'app:geometry',
+      exportProperty: 'Export',
+    },
+  },
+  coordinateSources: [
+    'plugin/pins/transformedCoordinate',
+    'plugin/pins/coordinatesAfterDrag',
+  ],
+  customHighlightStyle: {
+    stroke: {
+      color: '#FFFF00',
+      width: 3,
+    },
+    fill: {
+      color: 'rgb(255, 255, 255, 0.7)',
+    },
+  },
+}
+```
+
+</details>
+
 ##### gfi.gfiLayerConfiguration
```

Comment:
https://github.com/Dataport/polar/pull/158/commits/ef4b1610e74a80834240ead157834e5c259fea67

@dopenguin I wanted to add an example for `filterBy` and realized that I don't know what type a `clickPosition` is. Do you know a good example for that?

Also, I made `properties` optional as it has a default value.

Reply:
In `packages/plugins/Gfi/src/utils/requestGfiWms.ts:197-198` `filterBy` basically is used *like* a boolean; it is possible that it may be extended in the future so a string is fine.

Thus, your example is on point - only remove the TODO :D

Reply:
https://github.com/Dataport/polar/pull/158/commits/ea31a5a107f779918c8ebdf166b324ace189a211